drape/drape_tests/CMakeLists.txt

```diff
@@ -31,7 +31,12 @@ target_compile_definitions(${PROJECT_NAME}
   PRIVATE GTEST_DONT_DEFINE_TEST WITH_GL_MOCK
 )
 
-add_subdirectory(../../3party/googletest ${CMAKE_CURRENT_BINARY_DIR}/googletest)
+find_package(GTest "1.12.0")
```

Comment:
Probably QUIET option is needed to avoid log pollution when the package is not found, and it is found. What is the output in these both cases?

Reply:
The current output for the **not found** scenario is:
````
-- Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR GTEST_MAIN_LIBRARY) (Required is at least version "1.12.0")
````
When found it is:
````
-- Found GTest: /usr/local/lib/cmake/GTest/GTestConfig.cmake (found suitable version "1.12.1", minimum required is "1.12.0")
-- The build will rely on the system provided GoogleTest version

````